src/com/amazon/ion/benchmark/IonSchemaUtilities.java

```diff
@@ -45,6 +46,7 @@ public class IonSchemaUtilities {
     public static final String KEYWORD_MAX = "max";
     public static final String KEYWORD_SCALE = "scale";
     public static final String KEYWORD_PRECISION = "precision";
+    public static final String KEYWORD_VALID_VALUE = "valid_value";
```

Comment:
As per Ion schema specification `valid_values` is correct the constraint name. 
Reference : https://amzn.github.io/ion-schema/docs/spec.html#valid_values